.connector-store/meta.json

```diff
@@ -19,6 +23,28 @@
                 "MI 4.1.0",
                 "MI 4.0.0"
             ],
+            "operations": [
+                {
+                    "name": "init",
+                    "description": "configure Redis connector",
+                    "isHidden": false
```

Comment:
@ShammiL can this operations could be specific to a particular version? 
Just as an example lets say ping method is not supporting in a future version. How should we support it? isn't it better add the version and the product in the operations.

Reply:
Looping in @ChinthakaJ98

Reply:
+1 to add operations against a release

Reply:
In this approach the operations are added against the release. As per this metafile the given operations are only valid for the 3.1.3 version. If there is another release the relevant operations should be added separately for that.

Reply:
Ack.